packages/dashboard-core-plugins/src/controls/markdown/MarkdownUtils.ts

```diff
@@ -19,6 +22,16 @@ class MarkdownUtils {
 
     return title;
   }
+
+  static getClosedMarkdowns = memoize((closedPanels: ClosedPanels) =>
+    closedPanels
+      .filter(
+        panel =>
+          panel.component === 'MarkdownPanel' &&
+          panel.props.panelState.content !== ''
```

Comment:
Also need to filter out `props.panelState.content !== null`.

Right now if I open a Markdown, don't edit it, close it, then open another Markdown, it's showing the start page when it shouldn't.